gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/jdbc/EmbedStatement.java

```diff
@@ -1231,6 +1243,8 @@ else if (SanityManager.DEBUG) {
           checkRequiresCallableStatement(activation);
           activation.setFlags(activationFlags);
           activation.setFunctionContext(fnContext);
+					// TODO what if a different activation impl?
+					((GenericActivationHolder)activation).setConnProps(this.connProps);
```

Comment:
correct indentation to match above (perhaps copy from above to below)

Also why the need to copy the connProps to Activation? These should be maintained only on EmbedConnection and rest should get from that where required e.g. see userName in TransactionResourceImpl.